app/src/main/java/ir/kindnesswall/view/main/more/MoreFragment.kt

```diff
@@ -4,104 +4,195 @@ import android.os.Bundle
-        binding.blockedUsers.setOnClickListener { context?.let { BlockListActivity.start(it) } }
-        binding.showNumber.setOnClickListener { openPhoneVisibilityDialog() }
-        binding.contactUs.setOnClickListener { openSupportForm(requireContext()) }
-        binding.bugReport.setOnClickListener { openSupportForm(requireContext()) }
-        binding.suggestions.setOnClickListener { openSupportForm(requireContext()) }
-        binding.logInLogOut.setOnClickListener {
-            context?.runOrStartAuth {
-                showPromptDialog(
-                    getString(R.string.logout_message),
-                    positiveButtonText = getString(R.string.yes),
-                    negativeButtonText = getString(R.string.no),
-                    onPositiveClickCallback = {
-                        UserInfoPref.clear()
-                        AppPref.clear()
-                        KindnessApplication.instance.clearContactList()
-                        activity?.recreate()
-                    })
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        view as ComposeView
+        view.setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
+        view.setContent {
+            MdcTheme {
+                Scaffold(
+                    topBar = {
+                        TopAppBar(title = { Text(text = stringResource(R.string.more_items)) })
+                    }
+                ) {
+                    Column(
+                        modifier = Modifier
+                            .fillMaxSize()
+                            .verticalScroll(rememberScrollState())
+                    ) {
+
+                        val isLoggedIn = UserInfoPref.asLiveData(UserInfoPref::bearerToken)
+                            .observeAsState().value.isNullOrBlank().not()
+
+                        if (isLoggedIn)
+                            Header(
+                                name = UserInfoPref.asLiveData(UserInfoPref::name).observeAsState().value ?: "",
+                                avatarUrl = UserInfoPref.asLiveData(UserInfoPref::image).observeAsState().value ?: "",
+                                phoneNumber = UserInfoPref.asLiveData(UserInfoPref::phoneNumber).observeAsState().value
+                                    ?: ""
+                            )
+
+                        if (isLoggedIn)
+                            Item(
+                                textResId = R.string.my_profile,
+                                imgResId = R.drawable.ic_profile_placeholder_gary
+                            ) {
+                                UserProfileActivity.start(
+                                    requireContext(),
+                                    UserInfoPref.getUser()
+                                )
+                            }
+
+                        if (isLoggedIn)
```

Comment:
Why did you repeat the same `if` statement several times? AFAIK you can merge all in one `if` condition

Reply:
Yes, You're right. 

But I think this is more readable that add an if on 20 line higher that items and close scope them by more brackets.

Also in situations that some conditional items are not sequential, we must add a lot of like now